src/linear_algebra/general_linear_group.lean

```diff
@@ -71,3 +71,36 @@ rfl
 end general_linear_group
 
 end linear_map
+
+section
+
+variables {R M : Type*} [ring R] [add_comm_group M] [module R M] (T : M →ₗ[R] M)
+
+namespace linear_equiv
+
+/-- any invertible linear map can be written as a linear equivalence -/
+def of_invertible [invertible T] : M ≃ₗ[R] M :=
+linear_map.general_linear_group.to_linear_equiv (unit_of_invertible T)
+
+lemma coe_of_invertible [invertible T] :
+  ⇑(linear_equiv.of_invertible T) = T := rfl
+
+lemma coe_linear_map_of_invertible [invertible T] :
+  ↑(linear_equiv.of_invertible T) = T := by { ext, refl }
+
+@[simp] lemma of_invertible_symm_eq_inv_of [invertible T] :
```

Comment:
```suggestion
@[simp] lemma coe_linear_map_of_invertible_symm_eq_inv_of [invertible T] :
```
to distinguish it from the one below. You could drop the `eq_inv_of`s if you want shorter names.

Reply:
done